coalib/bears/Bear.py

```diff
@@ -460,14 +460,17 @@ def execute(self, *args, debug=False, **kwargs):
                 self.err('Bear {} violated one-based offset convention.'
                          .format(name), str(exc))
 
-            if self.kind() == BEAR_KIND.LOCAL:
-                self.err('Bear {} failed to run on file {}. Take a look '
-                         'at debug messages (`-V`) for further '
-                         'information.'.format(name, args[0]))
+            if (self.kind() == BEAR_KIND.LOCAL
+                    or self.section['log_level'].value == 'DEBUG'):
+                self.err(
```

Comment:
As @RaiVaibhav said at https://github.com/coala/coala/pull/5772/files#r223205682 , you have made changes in your PR which are meaningless.  Here you do

```diff
-self.err('Bear {} failed to run on file {}. Take a look '
+self.err(
+    'Bear {} failed to run on file {}. Take a look '
```

dont do that.

Dont touch other lines if you do not need to modify them.

It makes your patch bigger and harder to read, and therefore harder to review.

And your PR will be marked as a WIP because of this, because you havent yet provided a neat PR for review.